src/index.js

```diff
@@ -80,6 +81,7 @@ export default class Spinner extends React.Component {
 
   static defaultProps = {
     visible: false,
+    cancelable: true,
```

Comment:
@SandroMachado @niftylettuce  There are 2 main reasons I would vote against setting the default value to `false`:

1. By setting the default value to `false` you _will_ introduce a change since currently the spinner _is_ cancelable. Therefore users who will get the update and want to keep the same behavior as the current version would have to explicitly set `cancelable:true`.
2. The native Android Dialog is by default cancelable.

If you still insist, I can change it though.